modules/aws/config.go

```diff
@@ -0,0 +1,30 @@
+package aws
+
+import (
+	"log"
+	"os"
+)
+
+var customEndpoints map[string]string
+
+// SetAwsEndpointsOverrides function to override the aws endpoints globall one time
+func SetAwsEndpointsOverrides(endpoints map[string]string) {
+
+	if HasAwsCustomEndPoints() {
```

Comment:
Is this check actually required?  Since this variable is per-process and all tests run in a single process, it means that if there is a bug in localstack or moto and you want to use native AWS for a single test, there is no good way to do that.

Reply:
Yes. It is. The function just validate your are setting one time the custom endpoint. If you have a issue with a specific localstack endpoint you always could remove it from the custom endpoints array. I added some test checking this scenario

Reply:
Could you explain why it is required?  I'm actually confused on this point.

I understand you don't have to list a given service as a custom endpoint, but for example if I'm writing unit tests for an S3 module, and there is a bug in localstack for a single S3 API call, then all my tests would have to use AWS which defeats the whole purpose of this PR.

Alternatively, if it was re-modifiable, then it would be rather trivial to use localstack with one test suite and AWS with another suite.

Reply:
@synfinatic some historical info maybe could help to clarify https://github.com/gruntwork-io/terratest/pull/495#discussion_r415615658

Reply:
Fair enough.   As long as everyone understands that this global is for _all tests_ not just for a specific _test suite_.  The global is per-process and not per-suite (at least in the context of using `stretchr/testify/suite`)